app/adapters/-intl-adapter.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import IntlGetResult from 'ember-intl/models/intl-get-result';
+import Locale from 'ember-intl/models/locale';
+import IntlAdapter from 'ember-intl/adapter';
+
+function normalize (fullName) {
+    Ember.assert('Lookup name must be a string', typeof fullName === 'string');
+
+    return fullName.toLowerCase();
+}
+
+export default IntlAdapter.extend({
+    findLocale: function (locale) {
+        if (locale instanceof Locale) {
```

Comment:
What's `Locale` here, a class? If so can this be simplified since it is just data?